experimental/oak_baremetal_channel/src/server.rs

```diff
@@ -22,15 +22,15 @@ use anyhow::Context;
 /// and responds to them using the provided [`oak_idl::Handler`].
 pub fn start_blocking_server<H: oak_idl::Handler>(
     channel: Box<dyn Channel>,
-    mut oak_idl_service_implementation: H,
+    mut idl_service: H,
```

Comment:
I guess this is fine as an initial step, but I am not sure that we will always use the same IDL for all workloads, so this might have to be based on a different trait in the future.

Reply:
Isn't `oak_idl::Handler` already a generic trait that could be used by all future IDLs?

Reply:
It is somewhat generic as it just passes bytes back and forth, but assumes a very specific structure for the IDL. Specifically it assumes that is will there will always be pairs of requests and responses that map one to one and that the request will identify the method by a numeric method ID. It also does not support any additional metadata or anything like that. This will not always be suitable for all workloads we might want to run in future.